### PR TITLE
omicli should gracefully fail the request if auth fails 

### PR DESCRIPTION
**Pull request by yakman2020**
Fixes issue #194 
@Microsoft/omi-devs

### CODE REVIEW

File: Unix/etc/omicli.conf [hunk 1]
```diff
@@ -38,6 +38,6 @@
 ##
 ## loglevel,logpath,logfile -- set the loggiing options for MI client
 ##
-#loglevel = DEBUG
+loglevel = VERBOSE
 #logpath = var/log/
-#logfile = miclient.log
\ No newline at end of file
+#logfile = miclient.log
```
Comment (jeffaco) [inline Unix/etc/omicli.conf]: No longer a unix-format file, please fix.

File: Unix/wsman/wsmanclient.c [hunk 2]
```diff
@@ -143,12 +145,23 @@ static void HttpClientCallbackOnStatusFn2(
         const ZChar *text)
 {
     WsmanClient *self = (WsmanClient*) callbackData;
-    if (!self->enumerationState || self->enumerationState->endOfSequence)
+
+    
+    if (result == MI_RESULT_TIME_OUT)
```
Comment (jeffaco) [inline Unix/wsman/wsmanclient.c]: Brief comment on why you're translating to MI_RESULT_INVALID_PARAMETER

**Comment by jeffaco** [comment]
Please insure that the CI test failures on Linux are not due to this, and help figure out why regress is failing on CI. CI used to be the ONLY place regress worked!

File: Unix/etc/omicli.conf (hunk 1 above)
Comment (paulcallen) [inline Unix/etc/omicli.conf]: Should not check in this change

File: Unix/etc/omiserver.conf [hunk 3]
```diff
@@ -23,7 +23,7 @@
 ##
 ## loglevel -- set the log level of the server
 ##
-loglevel = DEBUG
+loglevel = VERBOSE
```
Comment (paulcallen) [inline Unix/etc/omiserver.conf]: This should be either disabled altogether, or WARNING should be specified in the checked in version

File: Unix/wsman/wsmanclient.c [hunk 4]
```diff
@@ -143,12 +145,23 @@ static void HttpClientCallbackOnStatusFn2(
...
+        result = MI_RESULT_INVALID_PARAMETER;
```
Comment (paulcallen) [inline Unix/wsman/wsmanclient.c]: should be MI_RESULT_FAILED WITH an OMI_Error object, and set probable cause to 111 - timeout.

**Pull request merged**